crates/asm-spec/asm.yml

```diff
@@ -102,6 +102,34 @@ StateRead:
                 - If there is no counter on the repeat stack.
                 - If there is no repeat registered to return to.
 
+            Reserve:
+              opcode: 0x0B
+              short: RES
+              description: |
+                Reserve space on the stack for `len` words.
+                The reserved space is set to 0.
+              stack_in: [len]
+            
+            Load:
+              opcode: 0x0C
+              short: LODS
+              description: Load the value at the given stack depth index relative to the bottom.
+              panics:
+                - Index is out of range.
+              stack_in: [index]
+              stack_out: [value]
```

Comment:
I was gonna say this is functionally equivalent to `DupFrom`, but just noticed `Load` indexes from the bottom of the stack while `DupFrom` is from the top. I can see both being useful.